hdf5.spec

```
@@ -15,13 +15,12 @@
 %global hdf5_major 1
 %global hdf5_minor 13
 %global hdf5_bugfix 0
-%global hdf5_prerelease rc5
```

Let's just comment this out so that future updates that once again are for a pre-release have this readily available to just set/define.

```suggestion
#global hdf5_prerelease rc5
```

---

fwiw this PR is just for testing which is why i did not request reviews and have not removed unneeded patch files.
real one will update to newer version once the new patch has landed and released in a hotfix-release and DAOS tests pass with this one.

---

should the comment be removed now?

---

Brian suggested to just comment this out in a previous remove of this